main.py

```diff
@@ -19,15 +19,16 @@ def main(site_data_path):
     global keywords
 
     # Load all for notesj data one time.
-    for f in glob.glob(site_data_path +"/*.yml"):
-
+    extensions = ["yml", "json"]
+    for f in chain(*[glob.glob(os.path.join(site_data_path, f"*.{ext}")) for ext in extensions]):
+        print(f)
```

Comment:
I think this part is still not fixed.
Only the .yml files are loaded in your script.

Reply:
I won't send a PR, I think this needs fixing @srush .